modules/ossm-config-control-plane-infrastructure-node-console.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-deployment-models.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-config-control-plane-infrastructure-node-console_{context}"]
+= Configuring all control plane components to run on infrastructure nodes using the web console
+
+Perform this task if all of the components deployed by the {SMProductShortName} control plane, including Istiod, Ingress Gateway, and Egress Gateway, along with optional elements, such as Prometheus, Grafana, and Distributed Tracing, run on infrastructure nodes. 
+
+If the control plane runs on a worker node, skip this task.
+
+.Prerequisites
+
+* The {SMProductName} Operator must be installed.
+* An account with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Create a project named `istio-system`.
++
+.. Navigate to *Home* -> *Projects*.
++
+.. Click *Create Project*.
++
+.. In the *Name* field, enter `istio-system`. The `ServiceMeshControlPlane` resource must be installed in a project that is separate from your microservices and Operators.
++
+These steps use `istio-system` as an example, but you can deploy your {SMProductShortName} control plane in any project as long as it is separate from the project that contains your services.
++
+.. Click *Create*.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Click the {SMProductName} Operator, then click *Istio Service Mesh Control Plane*.
+
+. Click the name of the control plane resource. For example, `basic`.
```

Comment:
Since the namespace is created in the preceding steps, the SMCP can't exist here. Therefore, the user should create it instead of editing it.

Reply:
Alternatively, you could remove the preceding steps so that the entire procedure talks about *modifying* an existing SMCP instead of creating one from scratch. I think this would be better, because you then don't need to explain everything else the user must set in the SMCP spec.

Reply:
Done, removed steps to create the SMCP.